requirements.txt

```diff
@@ -0,0 +1,12 @@
+# Runtime dependencies
+Flask==1.1.1
+Flask-API==1.1
+Flask-SQLAlchemy==2.4.1
+python-dotenv==0.10.3
+
+# Testing
+nose==1.3.7
+rednose==1.3.0
+pinocchio==0.4.2
+coverage==4.5.4
+pylint>=2.4.1
```

Comment:
This is not a big issue, but I notice that some dependencies have a higher version number now. Should we install the latest one?

This is the latest version I found for some of them:

```
Flask==1.1.2
Flask-API==2.0
Flask-SQLAlchemy==2.4.4
python-dotenv==0.14.0

coverage==5.3
```

Reply:
I picked up these from the Professor's template project, there might be a compatibility issue?

Reply:
I see. Then I think it is fine!